postgres-appliance/postgres_ha.sh

```diff
@@ -28,6 +28,9 @@ postgresql:
   listen: 0.0.0.0:${pg_port}
   connect_address: ${aws_private_ip}:${pg_port}
   data_dir: $PGDATA
+  pg_hba:
+  - hostssl all all 0.0.0.0/0 md5
+  - host    all all 0.0.0.0/0 md5
```

Comment:
It indeed has a security risk of someone running a internet-facing load balancer and no SSL, but the other side of the coin is when someone runs a server without SSL in the internal zone in VPC and will not able to connect, because there will be no entries in pg_hba.conf by default (hostssl will not be injected to the server running with SSL off). I think the second use-case will be more common in our environment.